.gitignore

```diff
@@ -25,3 +25,6 @@ dependency-reduced-pom.xml
 out.txt
 out.err
 inferred-predicates/
+
+drug-drug-interaction/
```

Comment:
Don't ignore these.
This is for your personal development situation, we don't want it to affect others.